Documentation for Cookie::fromString

ro0NL opened the pull request:
See #7278 

First time, be patient :)

There's also 2nd argument in `fromString` controlling URL decoding,  implying `Cookie::isRaw`, but i tend to leave it an implementation detail.

ro0NL committed:
Update http_foundation.rst

javiereguiluz commented on components/http_foundation.rst:
```diff
@@ -377,6 +377,14 @@ method takes an instance of
+You can also create a 
+:class:`Symfony\\Component\\HttpFoundation\\Cookie` object from a raw header value using :method:`Symfony\Component\HttpFoundation\Cookie::fromString`.
+
+.. tip::
```
I'm not sure about this TIP. Don't you think it's too low level? And it's not really a Symfony feature, but a PHP / HTTP feature, right?

HeahDude replied:
You're right, I'd say it's also a way to use symfony API, I guess it does not hurt to keep it, any other opinion on the matter?

ro0NL replied:
What about making the above a tip instead?

```
tip:
  You can create a :class:`Symfony\\Component\\HttpFoundation\\Cookie` object ...
```

javiereguiluz replied:
I still think we should not include this because:

* We don't do the same in other cases. We always use the OOP way of dealing with request/response, instead of going deep below to deal with HTTP headers directly.
* We must always be careful with all the examples we include because if we show *a way* to do things, some people will think that's *the way* to do things.

@xabbuh what's your opinion about this? Thanks!

HeahDude replied:
Those are strong arguments, thanks @javiereguiluz that's what I was asking for :).

javiereguiluz reviewed (commented):
👍

ro0NL committed:
feedback

ro0NL committed:
derp

ro0NL commented:
Updated.. removed the additional tip on low-level api usage; i agree with @javiereguiluz on this. 

So basically we simply mention `Cookie::fromString` now :)

HeahDude commented on components/http_foundation.rst:
```diff
@@ -377,6 +377,9 @@ method takes an instance of
 You can clear a cookie via the
 :method:`Symfony\\Component\\HttpFoundation\\ResponseHeaderBag::clearCookie` method.
 
+Note you can create a 
```
Missing a version added directive.

ro0NL replied:
Like so?

HeahDude replied:
Yes thanks!

ro0NL committed:
version added

HeahDude reviewed (approved):
👍

javiereguiluz committed:
Add method parenthesis

javiereguiluz commented:
Roland thanks for your first contribution to Symfony Docs and thanks for your great patience during the review process!

ro0NL commented:
No problems
